package.json

```diff
@@ -76,7 +76,7 @@
     "rollup-plugin-node-builtins": "2.1.2",
     "rollup-plugin-node-resolve": "4.0.0",
     "rollup-plugin-peer-deps-external": "2.2.0",
-    "rollup-plugin-postcss": "1.6.3",
+    "rollup-plugin-postcss": "2.0.0",
```

Comment:
sick of seeing that warning. this release allows us to use rollup > 1 without a warning 👍

Reply:
Nice!